packages/core/src/domain/configuration/configuration.ts

```diff
@@ -73,7 +77,8 @@ interface ReplicaUserConfiguration {
 export interface Configuration extends TransportConfiguration {
   // Built from init configuration
   beforeSend: GenericBeforeSendCallback | undefined
-  cookieOptions: CookieOptions
+  sessionStoreOptions: SessionStoreOptions
+  sessionStoreStrategyType: SessionStoreStrategyType | undefined
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about merging those?
we could have something more expressive like:
```typescript
SessionStoreOptions =
| { type: 'Cookie', cookieOptions: CookieOptions}
| { type: 'LocalStorage'}
| undefined
```

(edit) following thought:
what about introducing a concept that handle bridge as well?
Something like:
```typescript
SessionMode =
| { type: 'Cookie', cookieOptions: CookieOptions}
| { type: 'LocalStorage'}
| { type: 'Bridge'}
| undefined
```

Reply:
Merged as 
```javascript
type SessionStoreStrategyType = { type: 'Cookie'; cookieOptions: CookieOptions } | { type: 'LocalStorage' }
```

100% agree on the bridge - but let's do this in a follow-up PR.